partials/instagram.php

```diff
@@ -1,12 +1,40 @@
-<?php 
+<section id="instagram-feed" class="row"> 
+
+<?php
 $instagram_handle = IGV_get_option( '_igv_instagram_handle' ); 
-?>
-<section class="row"> 
+if ($instagram_handle) {
+  //echo '@' . $instagram_handle;
+  // Get instagram feed
+  $instagram_feed = get_instagram_feed($instagram_handle);
+
+  foreach($instagram_feed as $index => $instagram_item) {
+    $likes = $instagram_item->likes->count;
+    $comments = $instagram_item->comments->count;
+    $url = $instagram_item->images->standard_resolution->url;
+    $caption = $instagram_item->caption->text;
+  ?>
+
+  <?php if( $index != 0 ) { ?>
+    <div class="col s1">&#8200;</div>
```

Comment:
This is a "space" character. I was gonna ask u cuz the grid fails with empty divs (`.col`)